targets/csharp/source/PlayFabSDK/source/PluginContract.cs

```diff
@@ -3,6 +3,7 @@
     public enum PluginContract
     {
         PlayFab_Serializer,
-        PlayFab_Transport
+        PlayFab_Transport,
+        PlayFab_TransportWithPolly
```

Comment:
I don't think you need another PluginContract here. The PlayFab_Transport contract should be sufficient (and allows for easy swapping between the two).